spec/cocina/models/request_collection_spec.rb

```diff
@@ -1,228 +1,26 @@
-            "version": 3,
-            "description": {
-              "title": []
-            }
-          }
-        JSON
-      end
-
-      it 'has the attributes' do
-        expect(collection.attributes).to include(label: 'my collection',
-                                                 type: collection_type)
-        expect(collection.access).to be_kind_of Cocina::Models::Collection::Access
-        expect(collection.administrative).to be_kind_of Cocina::Models::Collection::Administrative
-        expect(collection.identification).to be_kind_of Cocina::Models::Collection::Identification
-        expect(collection.structural).to be_kind_of Cocina::Models::Collection::Structural
-      end
-    end
-
-    context 'with a full collection' do
-      let(:json) do
-        <<~JSON
-          {
-            "type":"#{collection_type}",
-            "label":"my collection",
-            "version": 3,
-            "access": {
-            },
-            "administrative": {
-              "hasAdminPolicy":"druid:mx123cd4567",
-              "releaseTags": [
-                {
-                  "who":"Justin",
-                  "what":"collection",
-                  "date":"2018-11-23T00:44:52Z",
-                  "to":"Searchworks",
-                  "release":true
-                },
-                {
-                  "who":"Other Justin",
-                  "what":"self",
-                  "date":"2017-10-20T15:42:15Z",
-                  "to":"Searchworks",
-                  "release":false
-                }
-              ]
-            },
-            "description": {
-              "title": [
-                {
-                  "primary": true,
-                  "titleFull":"my collection"
-                }
-              ]
-            },
-            "identification": {
-              "catalogLinks":[
-                {
-                  "catalog":"symphony",
-                  "catalogRecordId":"44444"
-                }
-              ]
-            }
-          }
-        JSON
-      end
-
-      it 'has the attributes' do
-        expect(collection.attributes).to include(label: 'my collection',
-                                                 type: collection_type)
-
-        expect(collection.administrative.hasAdminPolicy).to eq 'druid:mx123cd4567'
-
-        tags = collection.administrative.releaseTags
-        expect(tags).to all(be_instance_of Cocina::Models::ReleaseTag)
-
-        expect(collection.description.title.first.attributes).to eq(primary: true,
-                                                                    titleFull: 'my collection')
+  it_behaves_like 'it has collection attributes'
 
-        link = collection.identification.catalogLinks.first
-        expect(link.catalog).to eq 'symphony'
-        expect(link.catalogRecordId).to eq '44444'
-      end
-    end
+  it 'does not have Checkable model methods' do
+    item = described_class.new(required_properties)
```

Comment:
Do we need a test for something that is not present?  There are an infinite number of things this object does not do and they aren't important to test for.

Reply:
I can take it out;  I put it in to distinguish from Collection, but will remove.